### PR TITLE
Timer annotations based on sgd dataset

### PR DESCRIPTION
**Pull request by emilyjchang**
add some timer eval phrases based on google schema-guided-dialogue dataset

### CODE REVIEW

File: builtin/org.thingpedia.builtin.thingengine.builtin/eval/dev/annotated.txt [hunk 1]
```diff
@@ -1719,4 +1719,134 @@ UT: ontimer(date=$?) => @org.thingpedia.builtin.thingengine.builtin.alert();
 # log/106707
 U: set an alarm in 5 h from now
 UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
-UT: ontimer(date=[$now + 5h]) => @org.thingpedia.builtin.thingengine.builtin.alert();
+UT: ontimer(date=[$now + 5h]) => @org.thingpedia.builtin.thingengine.builtin.timer_expire();
```
Comment (gcampax) [inline builtin/org.thingpedia.builtin.thingengine.builtin/eval/dev/annotated.txt:1722]: Remind me, what's the difference between timer_expire and alert ?

**Comment by emilyjchang** [comment]
timer_expire will play a sound and also tell the user how much time has
passed. Alert will just play a sound (although I am thinking it might be
helpful to also tell the user what time it is now for the alert)

**Comment by gcampax** [comment]
How did we annotate each function in manifest.tt and dataset.tt? Are we able to distinguish between the two? It seems to me they overlap a lot even in how you annotated the dev set.

**Comment by emilyjchang** [comment]
The distinction should be when a user specifies that they would like a
timer (in x time) vs a alarm (alarm at x time).

**Comment by gcampax** [comment]
Yeah but how do we ensure that Genie doesn't generate the wrong combination? Because Genie will try to put timers to everything unless we control it in some way. I just want to make sure we are controlling how we generate. We can also eyeball a small synthetic dataset and look at what we generate...

**Comment by emilyjchang** [comment]
Yeah it does seem like from a user utterance perspective there can be some
ambiguity, since not all phrases are not going to be straightforward. I’m
afk until this evening, but I’ll look into it tonight and try to make
better distinctions. Then I think it would be helpful to do an initial
generation and see where we are, and improve from there.

**Comment by emilyjchang** [comment]
So I think the distinctions would just be, anything that requires a name / custom message will definitely go through say() -> reminder. Otherwise, "in" some amount of time with a unit will use timer_expire() and "at" some time will use alert(). I think the distinctions are made in these annotations. I can expand the utterances in dataset.tt and manifest.tt. But let me know if there is something specific that you find ambiguous.

**Comment by gcampax** [comment]
This looks good. Unit tests fail for unrelated reasons, so I'm going to merge this.

**Pull request merged**